Revised Pathpoint docs for V2 re-launch (#17972)
newrelic/docs-website: src/content/docs/new-relic-solutions/business-observability/install-pathpoint.mdx

freshnessValidatedDate: never
---

import installPathpoint from 'images/pathpoint_v2_screenshot_install_pathpoint.webp' 
  

> New Relic Pathpoint can be integrated with almost any New Relic account to provide comprehensive business process observability. To get started, it's important to have telemetry data that is relevant to your organization. This can include metrics, events, logs, or traces. While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 

Reviewer: ```suggestion
New Relic Pathpoint can be integrated with almost any New Relic account to provide comprehensive business process observability. To get started, it's important to have telemetry data that's relevant to your organization. This can include metrics, events, logs, or traces. While APM events and logs are a common starting point for Pathpoint, the choice of telemetry data is entirely up to you. 
```

<Step>

Go to <DNT>**[Apps & Visualizations](https://onenr.io/0yw40Vz6Ew3)**</DNT>.

</Step>

<Step>

> Search for  <DNT>**Pathpoint V2**</DNT> (you must specify `V2` to get the new codebase.  The older version has been deprecated).

Reviewer: ```suggestion
Search for  <DNT>**Pathpoint V2**</DNT> (you must specify `V2` to get the new codebase. The older version has been deprecated).
```

</Steps>

> Once you've installed Pathpoint, you can configure your stages, steps, and signals simply by using it's simple interactive UI.  You can also export and import JSON configurations which is useful for cloning and veresion controlling configurations as well as moving them between New Relic accounts. [Learn more about configuring Pathpoint](/docs/new-relic-solutions/business-observability/intro-pathpoint/). 

Reviewer: ```suggestion
Once you've installed Pathpoint, you can configure your stages, steps, and signals simply by using the interactive UI. You can also export and import JSON configurations, which is useful for cloning and version controlling configurations as well as moving them between New Relic accounts. [Learn more about configuring Pathpoint](/docs/new-relic-solutions/business-observability/intro-pathpoint/). 
```
